src/data/simple_dmatrix.cc

```diff
@@ -166,7 +166,7 @@ BatchSet<GHistIndexMatrix> SimpleDMatrix::GetGradientIndex(Context const* ctx,
   }
   if (!gradient_index_ || detail::RegenGHist(batch_param_, param)) {
     // GIDX page doesn't exist, generate it
-    LOG(INFO) << "Generating new Gradient Index.";
```

Comment:
Is there an issue with info log?

Reply:
It always gets printed in the test, which makes it harder to see when you are running the same test 1000 times. Is there another way to turn it off?